build.gradle.kts

```diff
@@ -585,8 +580,163 @@ tasks.register<Zip>("zip-src") {
     )
 }
 
-tasks.register("gradle-eclipse") {
-    group = "gradleBuildSystem"
-    description = "create eclipse project files"
-    dependsOn("clean-eclipse", "init", "resolve-test", "job", "eclipse")
+//test on variable
+tasks.register("print-test") {
+    doLast{
+        println("${project.properties["base.dir"]}src/java/")
+    }
+}
+
+eclipse{
+    description = "--> create eclipse project files"
+
+    // not sure if any of this from ant needed
+    // <pathconvert property="eclipse.project">
+    //   <path path="${basedir}"/>
+    //   <regexpmapper from="^.*/([^/]+)$$" to="\1" handledirsep="yes"/>
+    // </pathconvert>
+
+    // <taskdef name="eclipse"
+    //          classname="prantl.ant.eclipse.EclipseTask"
+    //          classpath="${build.dir}/lib/ant-eclipse-1.0-jvm1.2.jar" />
+    // <eclipse updatealways="true">
+    //   <project name="${eclipse.project}" />
+    //   <classpath>
+    //     <library path="${conf.dir}" exported="false" />
+    //     <library path="${basedir}/src/bin" exported="false" />
+    //     <library pathref="eclipse.classpath" exported="false" />
```

Comment:
Please see https://github.com/lewismc/nutch/blob/master/build.xml#L1158-L1170 and implement the equivalent using https://docs.gradle.org/current/dsl/org.gradle.plugins.ide.eclipse.model.EclipseClasspath.html